go/libraries/doltcore/sqle/sqlbatch_test.go

```diff
@@ -136,34 +137,42 @@ func TestSqlBatchInserts(t *testing.T) {
 	assertRowSetsEqual(t, expectedAppearances, allAppearanceRows)
 }
 
+func drainIter(iter sql.RowIter) error {
+	var returnedErr error
+	for {
+		_, err := iter.Next()
+		if err == io.EOF {
+			return returnedErr
+		} else if returnedErr != nil {
```

Comment:
Should this be `else if returnedErr == nil`? It starts as `nil` at first, so we'd never assign to it.

Reply:
Good catch.